app/check_tx.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 
 	"github.com/celestiaorg/celestia-app/x/blob/types"
+	blobtypes "github.com/celestiaorg/celestia-app/x/blob/types"
```

Comment:
[nit] we can remove the duplicate import statement if we always use the import alias in this file

```suggestion
	blobtypes "github.com/celestiaorg/celestia-app/x/blob/types"
```

Reply:
damn I though the linter would pick up on this!

Reply:
damn, we were doing this in a few places 1895fa48486c54a38acc50ada27c336e5f2c215b

Reply:
good catch!